src/core/data/grid/gridlayout.hpp

```diff
@@ -37,7 +37,10 @@ namespace core
     constexpr bool has_physicalQuantity_v = has_physicalQuantity<T>::value;
 
 
-    constexpr int centering2int(QtyCentering c) { return static_cast<int>(c); }
+    constexpr int centering2int(QtyCentering c)
```

Comment:
`constexpr static` ? 

don't see a class instance (constexpr or not) needed here

Reply:
I don't get the comment, it's just a free function why do you talk about class instances

Reply:
>  it's just a free function

yeah nvm